pages/common/date.md

```diff
@@ -26,3 +26,7 @@
 - Display the current date using the RFC-3339 format (`YYYY-MM-DD hh:mm:ss TZ`):
 
 `date --rfc-3339=s`
+
+- Set the current date using the format `MMDDhhmm[YYYY][.ss]`:
+
+`date {{09302359}}`
```

Comment:
That's because the `[YYYY][.ss]` part is optional. We could split it into two examples:
```Markdown
- Set the current date using the format `MMDDhhmm`:

`date {{09302359}}`

- Set the current date using the format `MMDDhhmmYYYY.ss`:

`date {{093023592021.59}}`
```

Reply:
That sounds like a good idea to aid clarity @marchersimon.